### PR TITLE
style added

### PR DESCRIPTION
**Pull request by PurAlex**

### CODE REVIEW

File: assets/js/ApisCalls.js [hunk 1]
```diff
@@ -27,4 +27,4 @@ async function makeCalls() {
   $("#lang").append(flagOfCountry, languageOfCountry, currencyOfCountry);
 }
 
-const button = $("#search").click(makeCalls);
+const button = $("#search").click(makeCalls, cityWeather); // I added function here - Pawel
```
Comment (OlgaFelczak) [inline assets/js/ApisCalls.js]: This comment at the end is unnecessary, please delete it.

File: assets/js/keys.js [hunk 2]
```diff
@@ -1,7 +1,9 @@
-const config = {
-  openWeatherApiKey: "9E92A9EV239V29V82V2",
-  xRapidAPIKey: "2870710a70mshb9dfd7b0cc1c38ap18047cjsn644f8c604a16",
-  xRapidAPIHost: "wft-geo-db.p.rapidapi.com",
-};
-
-var apiKey = "jykXqNRW0Y62n8S6SXW4C5ctkG3caX7l";
\ No newline at end of file
+var apiKey = "jykXqNRW0Y62n8S6SXW4C5ctkG3caX7l";
```
Comment (OlgaFelczak) [inline assets/js/keys.js]: What is this key for?

File: assets/js/keys.js [hunk 3]
```diff
@@ -1,7 +1,9 @@
...
+
+const config = {
+    openWeatherApiKey: "9E92A9EV239V29V82V2",
+    xRapidAPIKey: "2870710a70mshb9dfd7b0cc1c38ap18047cjsn644f8c604a16",
+    xRapidAPIHost: "wft-geo-db.p.rapidapi.com",
+};
+
+var APIkey = "11f628af2243468c36a25b569b68d689"; // Pawel's API key
```
Comment (OlgaFelczak) [inline assets/js/keys.js]: What is this key for?

**Review by OlgaFelczak — CHANGES_REQUESTED** [review]

File: assets/js/keys.js (hunk 2 above)
Comment (PurAlex) [reply assets/js/keys.js]: for the currency API

File: assets/js/keys.js (hunk 3 above)
Comment (PurAlex) [reply assets/js/keys.js]: Pawels key for the weather api

File: assets/js/keys.js (hunk 2 above)
Comment (OlgaFelczak) [reply assets/js/keys.js]: Ok, so please rename it so that it reflects this.

File: assets/js/keys.js (hunk 3 above)
Comment (OlgaFelczak) [reply assets/js/keys.js]: Ok, but you are merging this change, so again, please rename it so that it reflects this.

**Comment by OlgaFelczak** [comment]
Also, could you please double-check the code, because there are two search bars, and the background photo does not display on the whole width.

File: assets/js/keys.js (hunk 2 above)
Comment (PurAlex) [reply assets/js/keys.js]: ok

File: assets/js/keys.js (hunk 3 above)
Comment (PurAlex) [reply assets/js/keys.js]: ok

**Comment by OlgaFelczak** [comment]
Nicky, I can see two things right now:
- the search button is displayed weird
- you overwrote my changes, because when a city name is typed and searched then there is no paragraph with information from Wikipedia

**Review by OlgaFelczak — APPROVED** [review]

**Pull request merged**